### PR TITLE
Spring views contd

### PR DESCRIPTION
**Pull request by johanandren**
Now works (but needs proxy fix https://github.com/lightbend/kalix-proxy/pull/1600 )

### CODE REVIEW

File: sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ProtoDescriptorRenderer.scala [hunk 1]
```diff
@@ -32,7 +32,10 @@ object ProtoDescriptorRenderer {
       builder ++= " {\n"
       messageType.getFieldList.forEach { field =>
         builder ++= "  "
-        builder ++= field.getTypeName
+        if (field.hasTypeName)
+          builder ++= field.getTypeName
+        else
+          builder ++= field.getType.name().toLowerCase.drop(5)
```
Comment (octonato) [inline sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ProtoDescriptorRenderer.scala:38]: Just curious, but why this? And what is this drop 5?

File: sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ViewDescriptorFactory.scala [hunk 2]
```diff
@@ -164,15 +170,16 @@ private[impl] object ViewDescriptorFactory extends ComponentDescriptorFactory {
 
       // since it is a query, we don't actually ever want to handle any request in the SDK
       // the proxy does the work for us, mark the method as non-callable
-      (KalixMethod(annotatedMethod.copy(callable = false)).withKalixOptions(methodOptions), returnTypeDescriptor)
+      (
+        KalixMethod(annotatedMethod.copy(callable = false), methodOptions = Seq(methodOptions)),
+        queryInputSchemaDescriptor,
+        queryOutputSchemaDescriptor)
```
Comment (octonato) [inline sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ViewDescriptorFactory.scala:176]: thanks for the nice and readable formatting

**Review by octonato — APPROVED** [review]
LGTM

File: sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ProtoDescriptorRenderer.scala (hunk 1 above)
Comment (johanandren) [reply sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ProtoDescriptorRenderer.scala:38]: Primitive type name is always prefixed by `TYPE_`, this drops that prefix.

**Pull request merged**